src/main/resources/com/cloudbees/simplediskusage/QuickDiskUsage/index.jelly

```diff
@@ -33,7 +33,7 @@
             <j:choose>
                 <p>
                 <j:when test="${it.lastRun == 0}">
-                    not yet calculated, please <a href=".">try again in few seconds</a>
+                    not yet calculated, please <a href=".">try again in a few moments</a>
```

Comment:
a moment is a short period of time (usually seconds), o a distinct point in time.  
If you have multiple of them they will be disjoint - and as such I don't think this is correct :-)

"try again in a short while"? is possibly more what you are intending.